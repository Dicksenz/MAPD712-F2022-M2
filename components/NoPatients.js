import React from "react";

import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
import Icon from "@expo/vector-icons/MaterialCommunityIcons";

const NoPatients = ({ onPress }) => {
  return (
    <View style={styles.noPatientcontainerStyle}>
      <TouchableOpacity onPress={onPress}>
        <Icon name="refresh" color="#347174" size={50} />
      </TouchableOpacity>
      <Text style={styles.textStyle}>No patients found</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  noPatientcontainerStyle: {
    display: "flex",
    justifyContent: "center",
    alignItems: "center",
    height: "100%",
  },
  textStyle: {
    fontSize: 20,
  },
});

export default NoPatients;
