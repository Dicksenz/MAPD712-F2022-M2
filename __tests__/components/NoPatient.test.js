import React from "react";
import renderer from "react-test-renderer";

import NoPatients from "../../components/NoPatients";

test("Check if NoPatient component renders correctly", () => {
  const tree = renderer.create(<NoPatients />).toJSON();
  expect(tree).toMatchSnapshot();
});
