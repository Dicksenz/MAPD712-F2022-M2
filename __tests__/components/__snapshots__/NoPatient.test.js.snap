// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if NoPatient component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    No patients found
  </Text>
</View>
`;
